README.md

```diff
@@ -1,9 +1,23 @@
-# Sass Modules Migrator
+Easily upgrade your Sass stylesheets!
 
-This tool is designed to help migrate stylesheets from the old `@import`
-functionality to [the new module system][module-spec].
+<table>
+  <tr>
+    <td>
+      <img width="118px" alt="Sass logo" src="https://rawgit.com/sass/sass-site/master/source/assets/img/logos/logo.svg" />
+    </td>
+    <td valign="middle">
+      <a href="https://www.npmjs.com/package/sass-migrator"><img width="100%" alt="npm statistics" src="https://nodei.co/npm/sass-migrator.png?downloads=true"></a>
+    </td>
+    <td valign="middle">
+      <a href="https://travis-ci.org/sass/sass-migrator"><img alt="Travis build status" src="https://api.travis-ci.org/sass/sass-migrator.svg?branch=master"></a>
```

Comment:
The path for both of these URLs should be `sass/migrator`

Reply:
Done.